instrumentation/opentelemetry-instrumentation-requests/src/opentelemetry/instrumentation/requests/__init__.py

```diff
@@ -64,6 +66,7 @@
     _SUPPRESS_INSTRUMENTATION_KEY,
     http_status_to_status_code,
 )
```

Comment:
Could you include an example in the doc strings of how someone would be able to setup and export the duration metrics?

Reply:
Added example. Please have a look and suggest any changes needed in the example.

Reply:
np. Removed example.